index.html

```diff
@@ -24,6 +24,22 @@
     />
   </head>
   <body>
+    <div id="mobile_menu_container" class="mobile_menu hide">
+      <div class="close_container">
+        <img src="./img/cross.png" alt="" class="close">
+      </div>
+      <ul class="mobile_menu_content">
+        <li class="mobile_menu_items">
+          <a href="#work" class="mobile_menu_link" onclick="closeMenu()">Portfolio</a>        
+        </li>
+        <li class="mobile_menu_items">
+          <a href="#about" class="mobile_menu_link" onclick="closeMenu()">About</a>     
+        </li>
+        <li class="mobile_menu_items" >
+          <a href="#contact-section" class="mobile_menu_link" onclick="closeMenu()">Contact</a>     
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] [OPTIONAL] Please make sure that the navlinks work on desktop view as well.
https://github.com/bilalrajput09/My-Portfolio/blob/69df9bae73e3a11ff1502de8076c884167b760ac/index.html#L46-L51